### ISSUE TITLE
Add to interface spdlog::create method for a single sink

### ISSUE THREAD

**Issue by meldavis** [gabime/spdlog#224]
spdlog.h has a function `spdlog::create` for a vector of sinks.   Our  software has a log manager that maintains a single shared sink, similar to the 'dist_sink' that comes with spdlog. 

The log manager's function is: 
```c
logger manager::create_logger(std::string const& component_name)
{
	auto log = spdlog::get(component_name);

	if (log)
		return log;

	log = spdlog::create(component_name, shared_sink);
	return log;
}
```

However, the spdlog interface does not come with a method to add a single sink. If i use
`std::make_shared<spdlog::logger>(component_name, sink)` and then `spdlog::register_logger` to add it to the registry,  it does not get initialized with the global log_level. And I can't manually set the logger to the global log-level because the global log-level is not accessible.

Of course, I can accomplish this by directly calling into the spdlog::detail methods, but that's not very clean.

**Another option** would be to have the `spdlog::register_logger` method set the the logger's level to the global log-level.

**Comment by godbyk** [comment]
@meldavis This feature should be implemented by e16cb511e58a13f5a5c08d2504844fb66b047b97 if you'd like to give it a try.

**Comment by gabime** [comment]
Closed. Thanks @godbyk for the implementation (https://github.com/gabime/spdlog/commit/e16cb511e58a13f5a5c08d2504844fb66b047b97)
